Gives bogus `no-useless-constructor` errors

I created a TypeScript project with `oclif single` (from v1.17.0), which generated the following `.eslintrc`:

```
{
  "extends": [
    "oclif",
    "oclif-typescript"
  ]
}
```

In my code I have the following class:

```
class Foo {
  constructor(
    private bar: string,
  ) {}
}
```

`npm run posttest` says:

```
  25:3  error  Useless constructor  no-useless-constructor
```

referring to the above constructor. But it is not useless, as it initializes the object.

If I disable the `no-useless-constructor` rule and make a truly useless constructor, then I get this (correct) error:

```
  25:18  error  Unexpected empty constructor                  @typescript-eslint/no-empty-function
```

It seems that the oclif config erroneously leaves `no-useless-constructor` active. I tried removing `"oclif"` from the `extends` list, but that didn't help.

Thanks for oclif, and in particular for the TypeScript support!

--- a/index.js
+++ b/index.js
@@ -10,6 +10,8 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   plugins: ['@typescript-eslint', 'import'],
   rules: {
+    'no-useless-constructor': 'off',
+    '@typescript-eslint/no-useless-constructor': 'error',
     '@typescript-eslint/no-unused-vars': [
       'error',
       {

--- a/index.js
+++ b/index.js
@@ -10,17 +10,17 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   plugins: ['@typescript-eslint', 'import'],
   rules: {
-    'no-useless-constructor': 'off',
-    '@typescript-eslint/no-useless-constructor': 'error',
     '@typescript-eslint/no-unused-vars': [
       'error',
       {
         argsIgnorePattern: '^_',
       },
     ],
+    '@typescript-eslint/no-useless-constructor': 'error',
     '@typescript-eslint/no-var-requires': 'off',
     'import/no-unresolved': 'error',
     'no-unused-expressions': 'off',
+    'no-useless-constructor': 'off',
     'node/no-missing-import': 'off',
     'node/no-unsupported-features/es-syntax': 'off',
     'perfectionist/sort-classes': [

